newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -404,7 +407,7 @@ export const AssetDetails = ({
             <Line expand noMargin justifyContent="center">
               <BoxSearchResults
                 baseSize={128}
-                onRetry={fetchAssetsAndFilters}
+                onRetry={() => fetchAssetsAndFilters(showStagingAssets)}
```

Comment:
See my other comment, but I have a bad feeling if we start to have two places where the search options are (the context and here).
It means you have to ensure you don't call the method with the wrong boolean.